cmd/swarm/config.go

```diff
@@ -294,6 +320,28 @@ func envVarsOverride(currentConfig *bzzapi.Config) (config *bzzapi.Config) {
 		}
 	}
 
+	if storeDbPath := os.Getenv(SWARM_ENV_STORE_DB_PATH); storeDbPath != "" {
+		currentConfig.StoreParams.ChunkDbPath = storeDbPath
+	}
+
+	if storeDbCapacity := os.Getenv(SWARM_ENV_STORE_DB_CAPACITY); storeDbCapacity != "" {
+		if id, _ := strconv.Atoi(storeDbCapacity); id != 0 {
```

Comment:
This should return an error rather silently ignoring an incorrectly formatted value.

Reply:
Strictly for this config or all flags that try and convert string to int?

Reply:
I mean there should always be a good reason to ignore errors, and I don't feel like there is a good reason to here (e.g. if I put `--store.size=1O` (i.e. the letter O rather than a zero) then I'd rather get an error than silently ignore).

So I'd recommend all code like this to check the error, but I don't think it is necessary to make that change in this PR.

Reply:
Agreed. I’ll make the changes here and follow up with another PR addressing the others.

Reply:
It doesn't look like we need to use `os.Getenv` at all if we use the `EnvVar` property when defining our cli flags. https://github.com/ethersphere/go-ethereum/pull/195/files#diff-5be8f583a91eb8ca7024b8af84ef4b28R165 I think this also behaves as intended with the order of precedence being , cli -> env vars -> default. 

The most recent changes I see around this was in https://github.com/ethereum/go-ethereum/pull/15548.  @holisticode  I think we we might be able to get rid of `envVarsOverride` here as long as the `EnvVar` property is set when the flags are initially defined.

@lmars By using the correct types for our cli flags, we get syntax checking we want. 
Setting `--store.size=1O` would result in a in the following error.
`invalid value "1O" for flag -store.size: strconv.ParseUint: parsing "1O": invalid syntax`

Reply:
Indeed  it seems `EnvVar` can supply the needs of `envVarsOverride` - nevertheless before doing so thorough testing is required. I think to remember that `EnvVar` behavior with default values sometimes interfered with intended behavior, but I don't recall details.